pages/common/runit.md

```diff
@@ -1,7 +1,7 @@
 # runit
 
 > 3-stage init system.
-> More information: <https://wiki.archlinux.org/index.php/Runit>.
+> More information: <https://manned.org/runit>.
```

Comment:
This would be a more appropriate link here, since runit does actually have a homepage.
```suggestion
> More information: <http://smarden.org/runit>.
```

Reply:
Still waiting on #5790